.gitignore

```
@@ -27,5 +27,5 @@ yarn-error.log*
 /public/build/
 
 # prisma
-/prisma/dev.db
-.env
\ No newline at end of file
+/prisma/data.db
```

Did you change the generation of the DB anywhere? I still get `dev.db` when asking `prisma` to initialize a db for me.

Maybe safer would be:
```suggestion
/prisma/*.db
```